tilecloud_chain/server.py

```diff
@@ -92,7 +93,18 @@ def __init__(self):
         ]
 
         if self.cache["type"] == "s3":  # pragma: no cover
-            self.s3_client = tilecloud.store.s3.get_client(self.cache.get("host"))
+            error = None
+            success = False
+            for n in range(10):
+                time.sleep(n * 10)
+                try:
+                    self.s3_client = tilecloud.store.s3.get_client(self.cache.get("host"))
+                    success = True
```

Comment:
Missing break statement ? It seems it continue the for loop after success.

Reply:
Oups, thanks :-)